src/Foliage.rb

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env ruby
+
+class Foliage
+  @height
+  @offset
+
+  def initialize
+    @height = 0
+    @offset = 0
+  end
+
+  # Increases the height of the foliage by one
+  def grow
+    @height += 1
+    return self
```

Comment:
Need to document that self is returned

Reply:
https://github.com/Wall-Dough/pointless/pull/5/commits/5eec638bf0a795610e71f7f26e2b7d5fa6d95dd3